requirements/test.txt

```diff
@@ -2,6 +2,7 @@
 # These are for testing
 factory-boy==2.7.0
 fake-factory==0.5.7
+nose
```

Comment:
Right, this was added because we were accidentally running the tests inside the dist/ directory. I'll remove it.